upup/pkg/kutil/drain.go

```diff
@@ -0,0 +1,656 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kutil
+
+////
+// Based off of drain in kubectl
+// https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/drain.go
+////
```

Comment:
There code is lacking ... Removed dependencies, cleaned up crap. Only have some way to do it, fixed unit testing, and probably have 3 edge cases. Need to document.